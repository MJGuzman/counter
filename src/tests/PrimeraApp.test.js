import React from "react";
import { shallow } from "enzyme";
import PrimeraApp from "../PrimeraApp";

describe("Prueba en componente", () => {
  test("debe de mostrar el componente", () => {
    const whapper = shallow(<PrimeraApp saludo="Hola, Soy Goku" />);

    expect(whapper).toMatchSnapshot();
  });
});
