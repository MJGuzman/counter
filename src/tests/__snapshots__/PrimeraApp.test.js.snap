// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en componente debe de mostrar el componente 1`] = `
<div>
  <h1>
    Hola, Soy Goku
  </h1>
  <p>
    Estamos empezando con React!!
  </p>
</div>
`;
